src/resources/views/crud/inc/form_fields_script.blade.php

```diff
@@ -49,11 +49,13 @@ class CrudField {
 
         enable(e) {
             this.input.removeAttr('disabled');
+            this.input.trigger('backpack_field.enabled');
```

Comment:
Let's broaden this, so that we can do it for other methods/events if needed in the future. Instead of triggering & catching the event on the input, let's do it on the wrapper. We know the wrapper is there. We know it doesn't have events. We know we can easily catch events on it from fields, because all fields only have one wrapper. And it will also add clarity to fields which have multiple inputs.

Wouldn't you think @pxpm ?

Reply:
@pxpm bump - I just merged your PR changing the event names (🎉), but I was also suggesting moving the PLACE where the events are triggered. From the INPUT to the WRAPPER.

What do you think about that - good idea or bad?

Reply:
The idea does not seem bad, but what we have in the javascript it's the `element`, the input, not the wrapper. (I mean the javascript in initElementFunction).

It would required us to get the wrapper from the `element` and then listen to the events on the wrapper. 
I think it could be cleaner, but does it worth it ? Not sure it's something we should focus, and also we planned somewhere else to allow devs to disable the wrapper `wrapper => false` ... so yeah, I leave you to think about this and decide if we should do it or not. In my opinion I wouldn't do it. 

Cheers

Reply:
What?! Wrapper false?! I kind of remember talking about that, but don't remember doing it... if that's the case, then yeah... events on the wrappers don't really make sense.

Ok let's table this for now, we'll think about it again when it's all done.